cvat-ui/src/actions/about-actions.ts

```diff
@@ -0,0 +1,55 @@
+import { AnyAction, Dispatch, ActionCreator } from 'redux';
+import { ThunkAction } from 'redux-thunk';
+
+import getCore from '../core';
```

Comment:
The module was renamed in the latest patch.
So, it should be 
```
 import getCore from 'cvat-core'; 
```

It's a reason why UI doesn't work in your patch.